doom-modeline-segments.el

```diff
@@ -650,7 +650,7 @@ Uses `all-the-icons-octicon' to fetch the icon."
   (setq doom-modeline--vcs-icon
         (when (and vc-mode buffer-file-name)
           (let* ((backend (vc-backend buffer-file-name))
```

Comment:
One more question, should use `file-local-name` for `vc-backend` as well?

Reply:
Thanks for accepting the patch!

I don't think that would be a problem. https://github.com/emacs-mirror/emacs/blob/6430c8419c4bd007c45f7cd3abacbdcf4ad01401/lisp/vc/vc-git.el#L264-L267

Here, the prefix `(vc-git--literal-pathspec name)` is added *after extracting relatie path*.

While in `vc-state`, [the file is prefixed with the literal first](https://github.com/emacs-mirror/emacs/blob/6430c8419c4bd007c45f7cd3abacbdcf4ad01401/lisp/vc/vc-git.el#L354) and then [relative file name/path is retrieved](https://github.com/emacs-mirror/emacs/blob/6430c8419c4bd007c45f7cd3abacbdcf4ad01401/lisp/vc/vc-git.el#L1836-L1837).

Reply:
Fine. Thanks for you reply!